content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/list_driver_isos.rb

```diff
@@ -21,7 +21,7 @@ def main
                       values_hash[nil] = '-- no ISO datastore for provider --'
                     else
                       values_hash[nil] = '-- select drivers ISO from list --'
-                      provider.iso_datastore.iso_images.pluck(:name).grep(/toolsSetup/).each do |iso|
+                      provider.iso_datastore.iso_images.pluck(:name).grep(/tools.*setup/i).each do |iso|
```

Comment:
The iso images come from the RHV iso domain (only RHV providers may be selected in the target infra provider dropdown) and it contains ISOs of given naming convention. This convention alas has changed between 3.6 ({RHEV,oVirt}-toolsSetup) and 4.0 (rhev-tools-setup). 

We believe that this regex is the right balance between strictness and generality.